src/collector-config.yaml

```diff
@@ -74,6 +135,16 @@ service:
   extensions: [basicauth/github]
   pipelines:
 
+    logs/deployments:
```

Comment:
Will having these enabled by default open up people running this to any security concerns they need to be aware of? Will there be any errors when running this without having the github app? maybe this should be opt-in instead of opt-out.

Reply:
I agree, I think it'll be best to leave it as a commented out section like we have with some of the other components.  That way the "Bonus Section" is actually what it says it is